content/Video_Display.md

```diff
@@ -590,6 +590,18 @@ Whether the Window is displayed can be toggled using
 (See [above](#ff4a-wy-window-y-position-r-w-ff4b-wx-window-x-position-7-r-w)
 for the definition of "Window visibility".)
 
+::: tip Window Internal Line Counter
+
+The window keeps an internal line counter that's functionally similar to LY, and increments alongside it. However, it only gets incremented when the window is visible, as described [here](https://github.com/gbdev/pandocs/blob/develop/content/Video_Display.md#ff4a---wy-window-y-position-rw-ff4b---wx-window-x-position--7-rw). That is whenever:
```

Comment:
```suggestion
The window keeps an internal line counter that's functionally similar to LY, and increments alongside it. However, it only gets incremented when the window is visible, as described [here](#ff4a---wy-window-y-position-rw-ff4b---wx-window-x-position--7-rw). That is whenever:
```
This should only point to the anchor, not the GitHub render; otherwise it'll break in the HTML version.